zulip/integrations/log2zulip/log2zulip

```diff
@@ -42,10 +41,10 @@ def send_log_zulip(file_name, count, lines, extra=""):
     # type: (str, int, List[str], str) -> None
     content = "%s new errors%s:\n```\n%s\n```" % (count, extra, "\n".join(lines))
     zulip_client.send_message({
-        "type": "stream",
-        "to": "logs",
-        "subject": "%s on %s" % (file_name, platform.node()),
-        "content": content,
+        'type': 'stream',
+        'to': 'logs',
+        'subject': "%s on %s" % (file_name, platform.node()),
+        'content': content,
```

Comment:
why did you change quote style here?  There's no advantage to `'` over `"`, and it adds a lot of extra churn to this diff.

Reply:
Hm, I did it because I usually use single quotes for non-output strings. I guess I was a little over-confident.